### PR TITLE
Campaign document role #623

### PR DESCRIPTION
**Pull request by borislavstoychev**
## Screenshots:

| Before              | After              |
| ------------------- | ------------------ |
| [before screenshot] | [after screenshot] |

## Testing

**New environment variables**:

- `env var` : env var details

**New dependencies**:

- `dependency` : dependency details

**New dev dependencies**:

- `dependency` : dependency details

### CODE REVIEW

**Comment by borislavstoychev** [comment]
image
image
image
image
image
image
image
image
image
image
image

**Review by kachar — CHANGES_REQUESTED** [review]
@borislavstoychev Nice overall contribution!

I've added some note that would require manual changes regarding the links and reusing the routes.

One thing I don't get at all - why the whole PR is regarding the campaign document role and not for campaign document.

We add/delete documents and not roles

Can you please correct that everywhere so the role is not part of the PR

File: src/service/apiEndpoints.ts [hunk 1]
```diff
@@ -114,4 +114,13 @@ export const endpoints = {
     list: <Endpoint>{ url: '/company/list', method: 'GET' },
     viewCompany: (slug: string) => <Endpoint>{ url: '/company/' + slug, method: 'GET' },
   },
+  campaignDocument: {
+    listDocuments: <Endpoint>{ url: '/campaign-document-role', method: 'GET' },
```
Comment (kachar) [inline src/service/apiEndpoints.ts:118]: Can you send me the PR where we have the API for endpoint `/campaign-document-role`

Comment (kachar) [reply src/service/apiEndpoints.ts:118]: On the other hand we have backend for `/campaign-file`

Comment (borislavstoychev) [reply src/service/apiEndpoints.ts:118]: I guess I should delete the role from [here]( https://github.com/podkrepi-bg/api/pull/149) too

**Comment by kachar** [comment]
This PR has served its demo purpose so it can be closed now

**Pull request closed without merging**